### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/c405327eba2e159ad5ece966eb8c8b829d4eefc9

### CODE REVIEW

**Comment by amychurchwell** [comment]
@peiying2 this changes the string ID's on firefox/quantum which is breaking translations. Is there a way to update fluent ID's once they are shipped?

**Comment by peiying2** [comment]
> @peiying2 this changes the string ID's on firefox/quantum which is breaking translations. Is there a way to update fluent ID's once they are shipped?



> @peiying2 this changes the string ID's on firefox/quantum which is breaking translations. Is there a way to update fluent ID's once they are shipped?

You mean once the page is activated with the old ID? When the ID is changed, or the old string is removed because of the ID change, the locales that have been translated will be stored in Pontoon's translation memory. When the update shows up in Pontoon, a localizer can leverage previous translation. All these strings with ID change will appear as new strings in Pontoon.

File: en/firefox/browsers/chromebook.ftl [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+browsers-chromebook-dropdown-copy = Get { -brand-name-firefox } for { -brand-name-chromebook }
+
+# The span here is to display an underline underneath "Firefox"
+browsers-chromebook-why-get-firefox = Why get <span>{ -brand-name-firefox }</span> for { -brand-name-chromebook }?
+
+browsers-chromebook-while-a-chromebook = While a { -brand-name-chromebook } already has { -brand-name-chrome } installed, downloading and using { -brand-name-firefox } as your go-to browser provides you with a few benefits:
+
+browsers-chromebook-alwayson-tracking-protection = Always-on tracking protection: by default, { -brand-name-firefox } runs Enhanced Tracking Protection (ETP) to protect your personal data from known ad trackers, social trackers and cryptomining scripts that follow you around the web.
+
+browsers-chromebook-supporting-independent-technology = Supporting independent technology: since most major browsers are now running on { -brand-name-chromium }, there are fewer options that don’t feed into the Big Tech machine. { -brand-name-firefox } is backed by a non-profit and is dedicated to fixing the internet. Using { -brand-name-firefox } makes you a part of that.
```
Comment (peiying2) [inline en/firefox/browsers/chromebook.ftl:28]: `non-profit` -> `not-for-profit`. Do we want to add a noun after this?

File: en/firefox/browsers/chromebook.ftl [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+browsers-chromebook-is-it-hard = Is it hard to install { -brand-name-firefox } for { -brand-name-chromebook }?
+
+browsers-chromebook-we-wish-it = We wish it were more straightforward, but your { -brand-name-chromebook } would prefer to keep you in the { -brand-name-google-play } ecosystem. But, we think it’s worth the effort to install the { -brand-name-firefox } browser to your { -brand-name-chromebook } – and we’ve got resources to help you if you need them. There are two ways to get { -brand-name-firefox } on your device.
```
Comment (peiying2) [inline en/firefox/browsers/chromebook.ftl:32]: There are two `but` back to back in this sentence. Should we change the second one to `However, we think...`?

File: en/firefox/sync.ftl [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+start-syncing-in = Start syncing in four easy steps:
+
+firefox-sync-open-the-menu = Open the menu in the top right of { -brand-name-firefox } and select <strong>“Sign in to { -brand-name-sync }</strong>.”
+
+firefox-sync-click-get-started = Click <strong>“Get started”</strong> in the tab that opens.
+enter-an-email = Enter an email address and password to <strong>“Create a { -brand-name-firefox-account }.”</strong>
+
+firefox-sync-click-next-to = Click <strong>“Next”</strong> to get a verification sent to your email.
+
+firefox-sync-after-you-check = After you check your email and click the verification link. { -brand-name-firefox } will start automatically syncing in the background.
```
Comment (peiying2) [inline en/firefox/sync.ftl:33]: Replace the period with a comma between the two sentences.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Change requests here and there.

On the Quantum page, 20 locales are activated right now. The update will turn the pages to English (not because of redirect) because once activated, it takes manual step to deactivate it. If we decide to make the update, I will send communicate to the communities to make sure the page, for the locales already completed, is updated in a timely manner,

**Comment by peiying2** [comment]
Additional feedback on sweeping change to the string IDs is that all the history of discussion and pending suggestions will be wiped out. Only approved translations would show up in the Pontoon TMs. With that, we have decided to not to make the string ID change.

**Review by peiying2 — APPROVED** [review]
Thank you very much the fixes. It looks good now!

**Pull request merged**